docarray/utils/find.py

```diff
@@ -23,6 +26,10 @@ class _FindResult(NamedTuple):
     scores: AnyTensor
 
 
+class Doc(BaseDoc):
```

Comment:
why is this definition needed?

Reply:
This is used down in line 278. When the query data is in the form of AnyTensor, for incorporating parallelism, the queries are separated and then passed in a batch. The Doc class is created to store these individual queries.

Reply:
can you create another class with a private more descriptive name? Doc seems too generic

Reply:
yes, I will change it.